applications/SwimmingDEMApplication/custom_utilities/qsvmsdemcoupled_data.h

```diff
@@ -0,0 +1,99 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Joaquin Gonzalez-Usua
+//
+
+#if !defined(KRATOS_QSVMSDEMCOUPLED_DATA_H)
+#define KRATOS_QSVMSDEMCOUPLED_DATA_H
+
+#include "../FluidDynamicsApplication/fluid_dynamics_application_variables.h"
+#include "../FluidDynamicsApplication/custom_utilities/fluid_element_data.h"
+#include "../FluidDynamicsApplication/custom_utilities/qsvms_data.h"
+#include "utilities/element_size_calculator.h"
+#include "includes/checks.h"
+
+namespace Kratos {
+
+///@addtogroup SwimmingDEMApplication
+///@{
+
+///@name Kratos classes
+///@{
+
+template< size_t TDim, size_t TNumNodes, bool TElementIntegratesInTime = false>
+class KRATOS_API(SWIMMING_DEM_APPLICATION) QSVMSDEMCoupledData : public QSVMSData<TDim, TNumNodes, TElementIntegratesInTime>
+{
+public:
+
+///@name Type Definitions
+///@{
+
+using NodalScalarData = typename FluidElementData<TDim,TNumNodes, false>::NodalScalarData;
+using NodalVectorData = typename FluidElementData<TDim,TNumNodes, false>::NodalVectorData;
+
+///@}
+///@name Public Members
+///@{
+
+NodalScalarData FluidFraction;
+NodalScalarData FluidFractionRate;
+
+NodalVectorData FluidFractionGradient;
+NodalVectorData Acceleration;
+NodalVectorData BodyForce;
+
+double ElementSize;
+
+///@}
+///@name Public Operations
+///@{
+
+void Initialize(const Element& rElement, const ProcessInfo& rProcessInfo) override
```

Comment:
```suggestion
void Initialize(
    const Element& rElement,
    const ProcessInfo& rProcessInfo) override
```
Style guide (one argument per line in more than one argument methods)